python-package/README.rst

```diff
@@ -197,77 +271,76 @@ Install from `conda-forge channel <https://anaconda.org/conda-forge/lightgbm>`_
 
     conda install -c conda-forge lightgbm
 
-These are precompiled packages that are fast to install.
-Use them instead of ``pip install`` if any of the following are true:
+These packages support **CPU**, **GPU** and **CUDA** versions out of the box.
 
-* you prefer to use ``conda`` to manage software environments
-* you want to use GPU-accelerated LightGBM
-* you are using a platform that ``lightgbm`` does not provide wheels for (like PowerPC)
+**GPU**-enabled version is available only for **Windows** and **Linux** currently.
 
-For ``lightgbm>=4.4.0``, if you are on a system where CUDA is installed, ``conda install`` will automatically
-select a CUDA-enabled build of ``lightgbm``.
+**CUDA**-enabled version (since ``lightgbm>=4.4.0``) is available only for **Linux** currently and will be automatically selected if you are on a system where CUDA is installed.
 
-.. code:: sh
-
-    conda install -c conda-forge 'lightgbm>=4.4.0'
+|
 
 Install from GitHub
 '''''''''''''''''''
 
 All requirements from `Build from Sources section <#build-from-sources>`__ apply for this installation option as well.
 
-For **Windows** users, if you get any errors during installation and there is the warning ``WARNING:LightGBM:Compilation with MSBuild from existing solution file failed.`` in the log.
-
 .. code:: sh
 
     git clone --recursive https://github.com/microsoft/LightGBM.git
+    cd LightGBM
     # export CXX=g++-14 CC=gcc-14  # macOS users, if you decided to compile with gcc, don't forget to specify compilers
     sh ./build-python.sh install
 
 Note: ``sudo`` (or administrator rights in **Windows**) may be needed to perform the command.
+Run ``sh ./build-python.sh install --user`` to install into user-specific instead of global site-packages directory.
+
+Run ``sh ./build-python.sh install --no-isolation`` to assume all build and install dependencies are already installed, don't go to the internet to get them.
+
+|
 
 Run ``sh ./build-python.sh install --nomp`` to disable **OpenMP** support. All requirements from `Build Threadless Version section <#build-threadless-version>`__ apply for this installation option as well.
 
 Run ``sh ./build-python.sh install --mpi`` to enable **MPI** support. All requirements from `Build MPI Version section <#build-mpi-version>`__ apply for this installation option as well.
 
-Run ``sh ./build-python.sh install --mingw``, if you want to use **MinGW-w64** on **Windows** instead of **Visual Studio**. All requirements from `Build with MinGW-w64 on Windows section <#build-with-mingw-w64-on-windows>`__ apply for this installation option as well.
```

Comment:
Keep the same order as in the `Install from PyPI -> Build from Sources` section.